### PR TITLE
APP-723 Move IntegrationBridgeKeyProvider to a non-open source repo

### PR DESCRIPTION
**Pull request by robson-sanchez**
- Downgrade Log4j2 version to 2.5

### CODE REVIEW

**Review by ecarrenho-daitan — APPROVED** [review]
LGTM.

@dnsymphony please review.

File: pom.xml [hunk 1]
```diff
@@ -126,8 +126,27 @@
...
+            <version>${log4j2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
+            <version>${log4j2.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>jcl-over-slf4j</artifactId>
```
Comment (dnsymphony) [inline pom.xml]: Why do you need these if you are using log4j?

**Review by dnsymphony — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (robson-sanchez) [reply pom.xml]: Definitely, it's not required. I'll remove it.

Comment (robson-sanchez) [reply pom.xml]: DONE

**Review by dnsymphony — APPROVED** [review]

**Pull request merged**